### PR TITLE
Added version compatibility matrix

### PR DESCRIPTION
**Pull request by andreidan**

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -82,6 +82,16 @@ Manipulate BLOBs like so::
...
 
+Version matrix
+==============
++--------------+------------+
+| Crate Ruby   | CrateDB    |
++==============+============+
+| < 0.9        | < 0.57     |
++--------------+------------+
+| 0.9          | >= 0.57    |
++--------------+------------+
```
Comment (corck) [inline README.rst]: just to be sure. that table name change (  `schema_name` -> `table_schema`) was introduced in version 0.57?

Comment (andreidan) [reply README.rst]: @corck Yes, https://github.com/crate/crate/blob/0.57/CHANGES.txt#L224

**Comment by andreidan** [comment]
@corck tests seem to be failing because port 44200 is used already. Do you think it's ok to merge this and handle the testing environment as a separate issue?

**Comment by corck** [comment]
it's odd, it's working fine on my new pr #22, waiting for final review to merge that one. hope that will fix things and we can update your branch with that code.

**Comment by corck** [comment]
probably related to the sudo = false setting on travis, which I removed in that branch.

**Comment by corck** [comment]
@andreidan if you merge the latest master this should be green

**Pull request merged**